praxis-isncsci/ionic-app
# Ionic PWA infrastructure

Closes #1 

## Problem

- Add the basic Ionic app structure for us to import **ISNCSCI** libraries to
- Include the `isncsci` and `isncsci-ui` libraries

## Solution

- Created a `Ionic/Vue` project
- Imported the `isncsci` and `isncsci-ui` libraries
- Did initial setup of the ISNCSCI interface on the project's home file

## Testing

- [x] Ionic app runs

<details>
  <summary>Test case</summary>

  1. Open the project
  2. Run `npm install`
  3. Run `ionic serve`
  4. Confirm an Ionic app loads and the ISNCSCI grid is visible

</details>
